ibis/backends/tests/tpc/queries/duckdb/ds/05.sql

```diff
@@ -81,8 +81,8 @@ WITH ssr AS
      AND d_date BETWEEN cast('2000-08-23' AS date) AND cast('2000-09-06' AS date)
      AND wsr_web_site_sk = web_site_sk
    GROUP BY web_site_id)
-SELECT channel ,
-       id ,
+SELECT nullif(channel, '') AS channel ,
+       nullif(id, '') AS id ,
```

Comment:
This change, and the same change in `80.sql` are because of ClickHouse's obsession with never using null values. Apparently this includes the grouping keys for `rollup` and friends.